subprojects/build-init/src/test/groovy/org/gradle/buildinit/plugins/internal/BuildScriptBuilderGroovyTest.groovy

```diff
@@ -521,19 +521,18 @@ Alternatively:
         builder.create(target).generate()
 
         then:
-        assertOutputFile("""/*
- * This file was generated by the Gradle 'init' task.
- */
-
-repositories {
-    // Use jcenter
-    // 
-    // Alternatively:
-    // - Could use Maven central instead
-    // - Or a local mirror
-    jcenter()
-}
-""")
+        assertOutputFile("/*\n" +
```

Comment:
Why migrate away from a multiline string?

Reply:
See the comment - there is a significant trailing space in one of those lines. This becomes tricky because:
- it is not evident that it is there
- ideavim plugin (and possibly other editors, or even an auto code formatter) tend to strip the trailing spaces like this, making the test fail after the file is modified somewhere, which is annoying.